src/_component/recommendation/recommendation.pug

```diff
@@ -0,0 +1,29 @@
+-
+    var recTitle = "Recommendations";
+    
+
+#recommendation
+.recommendation
+    h1.recommendation__title=recTitle
+.recommendation__section
+    .space
+    button.youtube-button
+        a.link-access(href="https://www.youtube.com")
+            span.youtube-icon(class="my-cv-youtube")
+    .space
+    .space
+
+    .recommendation-wrapper
+        .recommendation-user
+            span.my-cv-user
+        .recommendation-detail
+            h1.recommendation-name Simona Radulescu
+            h2.recommendation-job Software Engineer - JavaScript
+    .space
+    .recommendation-box
+        p.recommendation-text Bianca is a highly creative and easy-going person to work with. He is multi-skilled, has a vast knowledge in web development in general and frontend development in particular with jaw-dropping CSS skills. In addition, he is proactive, self motivated, intelligent, keen to learn new technologies, always updating his knowledge/skills and keen to share his vast knowledge with any and all. It was a pleasure working with him. Bianca makes a great addition to any team! 
```

Comment:
Re-read the text recommendation - after the first sentence, it's about a boy.

Reply:
I changed. Thank you!